### ISSUE TITLE
access to socket.io.js?

### ISSUE THREAD

**Issue by LennartHennigs** [wperw/node-red-contrib-socketio#2]
Hi,
if I include your nodes in my project, is there a path where I can access `socket.io.js`?
I usually used and included`/socket.io/socket.io.js` on the client side of things, not on the server (=node-red) side.

Cheers
Lennart

**Comment by wperw** [comment]
Hello, by default you can find socket.io.js under `/ socket.io / socket.io.js` for the client side.
This path can also be configured using the socketio-config configuration node.

`True` writing on" serving the customer file js "and setting the desired path in" InputPath? "Properties.

Hope this helps

**Comment by LennartHennigs** [comment]
Hey,
thanks for the quick answer. I looked at the config node, and yes, the settings are as you described.
When accessing `http://192.168.42.1:1880/socket.io/socket.io.js` I still get:

Cannot GET /socket.io/socket.io.js

**Update:** I figured it out. When I checked the "Bind to Node-red Istance" checkbox it worked. I used to have a different port defined - since I ported the server from node (port 3000) to node-red. Using a custom port seemed to be the problem.

Cheers.
